api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -159,7 +159,11 @@ private FhirConstants() {
 	public static final String LOCATION_CONTACT_POINT_ATTRIBUTE_TYPE = "fhir2.locationContactPointAttributeTypeUuid";
 	
 	public static final String PROVIDER_CONTACT_POINT_ATTRIBUTE_TYPE = "fhir2.providerContactPointAttributeTypeUuid";
-	
+
+	public static final String LOCATION_TYPE_ATTRIBUTE_TYPE = "fhir2.locationTypeAttributeTypeUuid";
+
+	public static final String LOCATION_TYPE_SYSTEM_URL = "fhir2.locationTypeSystemUrl";
```

Comment:
I absolutely do not think this is a good idea to put in a global property. We should either have the system hard-coded or have a global property that refers to the concept source and have the mapping to the system URL in the `fhir_concept_source` table.